test/FluentMigrator.Tests/Integration/TestCases/ProcessorTestCaseSourceExcept.cs

```diff
@@ -0,0 +1,26 @@
+#region License
+// Copyright (c) 2024, Fluent Migrator Project
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+#endregion
+
+namespace FluentMigrator.Tests.Integration.TestCases;
+
+public class ProcessorTestCaseSourceExcept<TIgnore1>() : ProcessorTestCaseSource(true, typeof(TIgnore1));
```

Comment:
I just realized that I forgot the type constraints here, I think this should be enough to ensure generic arguments derive from ProcessorBase what to you think ? 
The ProcessorTestCaseSource constructor is protected by the way.

Reply:
Why not change the TIgnore1 stuff to be TMigrationProccesor1 where TMigrationProcessor : ProcessorBase

Reply:
Yep that's what I did, just not with the same name